### PR TITLE
Added parameters to constructors

### PR DESCRIPTION
**Pull request by kopaygorodsky**
Added possibility to set parameters in Notification constructor

### CODE REVIEW

File: Entity/Notification.php [hunk 1]
```diff
@@ -45,11 +45,14 @@
      */
     protected $createdAt;
 
-    public function __construct()
+    public function __construct(string $title, string $message, array $recipients)
     {
         $this->id              = Uuid::uuid4();
         $this->recipientsItems = new ArrayCollection();
         $this->createdAt       = new \DateTime();
+        $this->setTitle($title);
```
Comment (akovalyov) [inline Entity/Notification.php]: `$this->title = $title`?

Comment (kopaygorodsky) [reply Entity/Notification.php]: It was like this, but I changed it.  I was thinking if I add some logic to setter...

File: Entity/Notification.php [hunk 2]
```diff
@@ -45,11 +45,14 @@
...
+        $this->setMessage($message);
+        array_walk($recipients, function ($recipient) {$this->addRecipient($recipient);});
```
Comment (akovalyov) [inline Entity/Notification.php]: something with codestyle.
Btw, does `this` work in the scope of lambda function? I am not sure about it -- https://3v4l.org/BK2dp

Comment (kopaygorodsky) [reply Entity/Notification.php]: image

Comment (kopaygorodsky) [reply Entity/Notification.php]: works

**Review by akovalyov — APPROVED** [review]

**Pull request merged**